nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/utils/FileExtensionFilterType.java

```diff
@@ -0,0 +1,33 @@
+package org.nmrfx.processor.gui.utils;
+
+import javafx.stage.FileChooser.ExtensionFilter;
+import org.nmrfx.processor.datasets.vendor.rs2d.RS2DData;
+
+import java.util.List;
+
+/**
+ * Enum to keep file chooser extension descriptions consistent for each set of extension filter.
+ */
+public enum FileExtensionFilterType {
+    ALL_FILES("All Files", "*.*"),
+    NMR_FID("NMR Fid","fid", "ser", "*.nv", "*.dx", "*.jdx", "*.jdf", "*.dat"),
+    NMR_DATASET("NMR Dataset", "*.nv", "*.ucsf", "*.dx", "*.jdx", "1r", "2rr", "3rrr", "4rrrr", RS2DData.DATA_FILE_NAME),
```

Comment:
do we want .dat for NMR_DATASET too?

Reply:
RS2DData.DATA_FILE_NAME is a constant for `data.dat`
Both the RS2D FID files and the dataset files require the filename to be `data.dat` or they won't be loaded.  So the filter type for both NMR_FID and NMR_DATASET should probably be the same. Either `*.dat` or `data.dat`?  @brucejohnson or @vfiack could confirm which one to use?

Reply:
I'd use data.dat, as I don't think there are any other files that end .dat.